src/App.js

```diff
@@ -1,17 +1,19 @@
-import Header from './components/Header/Header'
+import Header from './components/Header/Header';
 import Layout from './components/Layout/Layout';
-import FirstContextProvider from './contexts/FirstContext';
+import DataContextProvider from './contexts/DataContext';
+import InputContextProvider from './contexts/InputContext';
 
 function App() {
-  
-  return (
-    <>
-      <Header />
-      <FirstContextProvider>
-        <Layout />
-      </FirstContextProvider>       
-    </>
-  );
+	return (
+		<>
```

Comment:
we should switch the tab spacing from 8 spacing to two or it becomes harder to read nested statements as they scroll off github on smaller screens